tests/unit/progress/test_queries.py

```diff
@@ -0,0 +1,46 @@
+from unittest.mock import create_autospec
+
+import pytest
+
+from databricks.labs.ucx.framework.owners import Ownership
+from databricks.labs.ucx.framework.utils import escape_sql_identifier
+from databricks.labs.ucx.progress.queries import QueryProblemProgressEncoder
+from databricks.labs.ucx.source_code.queries import QueryProblem
+
+
+@pytest.mark.parametrize(
+    "query_problem, failure",
+    [
+        (
```

Comment:
inline to a variable within the function - it makes no sense to have a single-case paramatrised test.